tests/testcases/config-variables/src/lib.rs

```diff
@@ -10,17 +10,23 @@ fn handle_request(req: Request) -> Result<Response> {
     let query = req
         .uri()
         .query()
-        .expect("Should have a password query string");
+        .expect("Should have a expected_config_value query string");
     let query: std::collections::HashMap<String, String> = serde_qs::from_str(query)?;
-    let provided_password = query
-        .get("password")
-        .expect("Should have a password query string");
-    let expected_password = config::get("password")?;
+    let expected_config_value = query
+        .get("expected_config_value")
```

Comment:
If you change this, I think you need to change the test that passes the query string?  It might be easier to leave the actual query string as was, as you did for the config key.  It's only the variables and the message that I found confusing - sorry if I was overemphatic in my suggestions for change.